svf-llvm/lib/extapi.c

```diff
@@ -23,291 +22,209 @@ void *SyGetmem(unsigned long size)
     return NULL;
 }
 
-__attribute__((annotate("ALLOC_RET")))
-__attribute__((annotate("STATIC")))
 const unsigned short **__ctype_b_loc(void)
 {
-    return NULL;
+    return malloc(10);
```

Comment:
why malloc(10)? better to have a macro rather than a concrete number.

If you remove all STATIC functions, how about static functions mentioned here:
https://github.com/SVF-tools/SVF/issues/1186